src/date/expand-pattern.js

```diff
@@ -24,8 +24,30 @@ define([
  * - { datetime: "full" } returns "EEEE, MMMM d, y 'at' h:mm:ss a zzzz";
  * - { pattern: "dd/mm" } returns "dd/mm";
  */
+
 return function( pattern, cldr ) {
-	var result;
+	var result, getDateTime, skeleton, dateSkeleton = "", timeSkeleton = "";
+
+	getDateTime = function( type, dateSkeleton, timeSkeleton, cldr ) {
+		var result;
+		result = cldr.main([
+			"dates/calendars/gregorian/dateTimeFormats",
+			type
+		]);
+		if ( result ) {
+			result = formatMessage( result, [
+				cldr.main([
+					"dates/calendars/gregorian/dateTimeFormats/availableFormats",
+					timeSkeleton
+				]),
+				cldr.main([
+					"dates/calendars/gregorian/dateTimeFormats/availableFormats",
+					dateSkeleton
+				])
+			]);
+		}
+		return result;
+	};
```

Comment:
Yes sir we can. But that would also lead to increase in size of these lines which I thought has more overhead:

https://github.com/ManrajGrover/globalize/blob/fix-272-handleDateTime/src/date/expand-pattern.js#L67
https://github.com/ManrajGrover/globalize/blob/fix-272-handleDateTime/src/date/expand-pattern.js#L69
https://github.com/ManrajGrover/globalize/blob/fix-272-handleDateTime/src/date/expand-pattern.js#L71
https://github.com/ManrajGrover/globalize/blob/fix-272-handleDateTime/src/date/expand-pattern.js#L73

Surely we can add two more arguments `datePath` and `timePath` if that is what is wanted.